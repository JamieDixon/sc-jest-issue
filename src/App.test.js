import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import Foo from './components';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
    const comp = renderer.create(<Foo>Hello</Foo>);
    const tree = comp.toJSON();
    expect(tree).toMatchSnapshot();
});
