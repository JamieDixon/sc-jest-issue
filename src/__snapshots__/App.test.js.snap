exports[`test renders without crashing 1`] = `
<div
  className="sc-bdVaJa fmDBXD">
  Hello
</div>
`;
